### ISSUE TITLE
@doesNotPerformAssertion added when expectException is used 

### ISSUE THREAD

**Issue by DaveLiddament** [rectorphp/rector#2210]
| Subject        | Details                                                      |
| :------------- | :----------------------------------------------------------- |
| Rector version | Rector v0.5.19          |
| PHP version    | PHP 7.3.5-1+0~20190503093827.38+stretch~1.gbp60a41b (cli) (built: May  3 2019 09:38:28)                               |
| Full Command   | vendor/bin/rector process -s=phpunit60 -c rector.yaml  | 

## Current Behaviour 

Output after running rector:

```diff

+    /**
+     * @doesNotPerformAssertion
+     */
     public function testInvalidType(): void
     {
-        $this->setExpectedException(InvalidArgumentException::class);
+        $this->expectException(InvalidArgumentException::class);
         new CustomerType("foo");
     }
 }

```

## Minimal PHP Code Causing Issue
```php
<?php 
class FooTest extends \PHPUnit\Framework\TestCase
 {
     public function testInvalidType(): void
     {
        $this->expectException(InvalidArgumentException::class);
         new CustomerType("foo");
     }
 }

```

## Expected Behaviour

Code should **NOT** have the `@doesNotPerformAssertion` added because  `$this->expectException(InvalidArgumentException::class);` is an assertion.

**Comment by TomasVotruba** [comment]
This fix is easy, as it already checks for `expectException`.
Would you like to try to tfix that or shall I?

**Comment by DaveLiddament** [comment]
I'll take a look

**Comment by TomasVotruba** [comment]
Hint: https://github.com/rectorphp/rector/blob/e72eb34a4c0d045c781299fb5c8624568841b0a8/packages/PHPUnit/src/Rector/ClassMethod/AddDoesNotPerformAssertionToNonAssertingTestRector.php#L281

**Comment by DaveLiddament** [comment]
Proposed fix for #2217

**Comment by TomasVotruba** [comment]
Closed by #2217
